frontend/src/metabase/visualizations/lib/LineAreaBarRenderer.js

```diff
@@ -72,10 +72,12 @@ import type { VisualizationProps } from "metabase/meta/types/Visualization";
 const BAR_PADDING_RATIO = 0.2;
 const DEFAULT_INTERPOLATION = "linear";
 
-const UNAGGREGATED_DATA_WARNING = col =>
-  t`"${getFriendlyName(
+const UNAGGREGATED_DATA_WARNING = col => ({
+  key: UNAGGREGATED_DATA_WARNING,
```

Comment:
I think `key` should be a string rather than the function itself.

Also for a bit of cleanup, maybe move these all to frontend/src/metabase/visualizations/lib/warnings.js and make them all functions, following this form (with or without the `create` prefix):

```
export const UNAGGREGATED_DATA_WARNING = "UNAGGREGATED_DATA_WARNING";

export function unaggregatedDataWarning(col) {
  return {
    key: UNAGGREGATED_DATA_WARNING,
    text: ...
  }
}
```

Reply:
whoops! I meant to make it a string but forgot quotes. 

Consolidating sounds good!